src/utils/formHelperFunctions.ts

```diff
@@ -98,7 +99,7 @@ const lookUpTrigger = (step: any, elementID: any, elementType: any) => {
   } else return {};
 };
 
-const nextStepKey = (nextConditions: any, metadata: any) => {
+const nextStepKey = (nextConditions: any, metadata: any, fieldValues: any) => {
```

Comment:
fieldValues is globally available so it doens't have to be passed in, just import it into the file